strawberryfields/gbs/vibronic.py

```diff
@@ -145,27 +159,37 @@ def gbs_params(
     >>>     ]
     >>> )
     >>> d = np.array([0.2254, 0.1469, 1.5599, -0.3784, 0.4553, -0.3439, 0.0618])
-    >>> p = gbs_params(w, wp, Ud, d)
+    >>> T = 0.0
+    >>> p = gbs_params(w, wp, Ud, d, T)
 
     Args:
         w (array): normal mode frequencies of the electronic ground state (:math:`\text{cm}^{-1}`)
         wp (array): normal mode frequencies of the electronic excited state (:math:`\text{cm}^{-1}`)
```

Comment:
maybe, to make the description as general as possible, you may want to refer to ``w`` and ``wp`` as the normal mode frequencies of the initial and final electronic states. Such a description would work for vibronic spectrum computed for transitions between (electronic) ground and excited states of the same molecule and between electronic ground states of a molecule wit different charges.

Reply:
done.